toolbox/linux/index.rst

```diff
@@ -0,0 +1,32 @@
+Linux
+=====
+
+`Linux <https://en.wikipedia.org/wiki/Linux>`_ is an `Operating System
+<https://en.wikipedia.org/wiki/Operating_system>`_, like Windows or Mac OS X
+[#fn-kernel]_. Unlike those operating systems, Linux is open source, and
+completely free of charge. While you may expect something that costs no money
+and is developed by a team of "volunteers"[#fn-vol]_ to be low quality, Linux
+is actually very good.
+
+If you follow through this subsection, you should be at the point where you have
+installed Linux on your computer or a virtual machine. This is a prerequisite
+for a number of things in the "Hacker's Toolbox."
+
+HacSoc typically holds Linux install fests at the beginning of each year. These
+are excellent times to get help on this stuff!
```

Comment:
Suggestion: add subsection "What If I Missed the Install-Fest" (especially since one *just* happened)